content/shmem_team_my_pe.tex

```diff
@@ -1,41 +1,41 @@
 \apisummary{
-    shmem\_team\_my\_pe returns the calling process's virtual rank in the
-    provided team.
+    Returns the number of calling \ac{PE} within the provided team.
 }
 
 \begin{apidefinition}
 
 \begin{Csynopsis}
-int @\FuncDecl{shmemx\_team\_my\_pe}@(shmem_team_t newteam);
+int @\FuncDecl{shmem\_team\_my\_pe}@(shmem_team_t team);
 \end{Csynopsis}
 
 \begin{apiarguments}
-\apiargument{IN}{newteam}{A valid SHMEM team handle.}
+\apiargument{IN}{team}{A valid SHMEM team handle.}
 \end{apiarguments}
 
 \apidescription{
-The shmemx\_team\_my\_pe function returns the calling process's virtual
-rank in the provided team. The rank will be a value between 0 and N-1,
-for a team of size N. Different members of a team cannot have the same
-rank. For the team SHMEM\_TEAM\_WORLD, this will return shmem\_my\_pe.
+The \FUNC{shmem\_team\_my\_pe} function returns the number of calling \ac{PE} within the
+provided team. The number will be a value between 0 and N-1,
+for a team of size N. Each member of the team has a unique number.
+For the team \CONST{SHMEM\_TEAM\_WORLD}, this will return the same value
+as \FUNC{shmem\_my\_pe}.
 
 Error checking will be done to ensure a valid team handle is provided.
 All errors are considered fatal, and will result in the job aborting
 with an informative error message.
```

Comment:
See earlier comment on error handling. Here, I think a return value of -1 is an acceptable error sentinel.